README.rst

```diff
@@ -80,6 +85,30 @@ having local documentation, clone the repository and install the package in
     python3 setup.py develop --user
 
 
+Mac OS X
```

Comment:
I think it's called macos these days

Reply:
@dekuenstle  you raise an interesting point here. As you commented this on different files, I opened an Issue (#125) to discuss this in general.